website/frontend/templates/docs/scripting.html

```diff
@@ -611,6 +611,16 @@ <h3 id="len_text">$len(text)</h3>
           <li>Returns the number of characters in <code>text</code>.</li>
         </ul>
 
+        <h3 id="lenmulti_name_separator">$lenmulti(name,separator="; ")</h3>
+
+        <ul>
+          <li>Returns the number of elements in the multi-value tag
+            <code>name</code>. A literal value representing a multi-value can
+            be substituted for <code>name</code>, using the
+            <code>separator</code> (or "; " if not passed) to coerce the value
+            into a proper multi-valued tag.</li>
```

Comment:
I have the feeling we should have an example here :) I guess many will have a hard time understanding this

Reply:
Is this example okay, or should I include something like `$lenmulti(%artists%)` will return "3" for release https://musicbrainz.org/release/8c759d7a-2ade-4201-abc2-a2a7c1a6ad6c

Reply:
I think that works fine